orchestrator.py

```diff
@@ -144,6 +143,8 @@ def on_message(self, client, userdata, msg):
             self.currently_logging = True
         elif topics.WirelessModule.all().stop.matches(msg.topic):
             self.currently_logging = False
+        elif msg.topic == topics.Camera.flip_video_feed:
+            config.set_rotation(config.read_configs().get(config.ROTATION_KEY, 0))
```

Comment:
Unless I'm missing something, this doesn't actually change the rotation, right?

Reply:
Forgot to put the formula in
bd91d73